h1st/core/model.py

```diff
@@ -43,6 +43,33 @@ def train(self, prepared_data):
            my_model_2.load('1st_version')
     """
 
+    @property
+    def model(self) -> Any:
+        return getattr(self, '__model', None)
+
+    @model.setter
+    def model(self, value: Any):
+        setattr(self, '__model', value)
+
+    @property
+    def metrics(self) -> Any:
```

Comment:
What is `model.metrics`???

Consider the fact that `Model` is a very fundamental, busy base class. Thus, when you add any attribute or function to it, you have to be sure that it's very broadly fundamental and meaningful. You don't just anything you like or think you need at the moment. Concepts like model metrics or statistics are either ephemeral or subject to change on a version by version basis. You may be better off grouping these under some more general property. Let's discuss how best to design around your functional needs.

Reply:
I think metrics is fundamental property of model. Maybe, I don't understand your deep thought. We need a discussion.